config/config.schema.json

```diff
@@ -1551,9 +1551,6 @@
     "kusto": {
       "type": "object",
       "properties": {
-        "subscription": {
```

Comment:
i had this introduced on friday but realized we don't need it when we go for the global subscription